src/rabbit_mgmt_db_handler.erl

```diff
@@ -42,8 +42,16 @@ add_handler() ->
 gc() ->
     erlang:garbage_collect(whereis(rabbit_event)).
 
+%% some people have reasons to only run with the agent enabled:
+%% make it possible for them to configure key management app
+%% settings such as rates_mode.
+get_env(Key) ->
+    application:get_env(
+      rabbitmq_management, Key,
+      application:get_env(rabbitmq_management_agent, Key)).
```

Comment:
`application:get_env/3` is only supported in Erlang R16. You can only rely on `application:get_env/2`.